students/JasneetChandok/Lesson04/mailroom2_v2.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3.6
+
+'''Lesson 3 - Mailroom Part 2'''
+
+# Importing Systems Module
+import sys
+
+
+# Starter list of existing donors
+donors = {'jasneet paramjit': [50, 200, 500],
+          'simran kaur': [100, 200, 500],
+          'manikaran chandok': [100, 200, 500, 680],
+          'manveen chandok': [10, 200, 65],
+          'sanjam patwalia': [200, 50]}
+
+
+# List of existing donor
+def existingdonors_list():
+    donor_list = "\n ".join(donor for donor in donors)
+    print("\nList of Existing Donors:", "\n", donor_list.title(), "\n")
+
+
+# Add donation to an existing donor
+def add_donation():
+        donor_selection = input("Enter 'Full Name' of the donor >")
+        donor_donation = input("\nEnter the 'Amount' donated >")
+        donor = donor_selection.strip().lower()
+        if donor in donors:
+            donor_amount = donors.get(donor)
+            donor_amount.append(int(donor_donation))
+            print(f'\n Donation amount of "{donor_donation}" has been recorded for {donor.title()}!!\n')
+            print('''Thank you note to share out with the donor below:\n
+              Dear {}s
+
+              Thank you for your very kind donation of ${:.2f}.
+              It will be put to very good use.
+
+                             Sincerely,
+                                -The Team
+                '''.format(donor.title(), int(donor_donation)))
+        else:
+            print('\nDonor mentioned does not exist in the list!!')
+
+
+# Create a record for a new donor
+def new_donor():
+        donor_name = input("Enter 'Full Name' of the donor >")
+        donor_name = donor_name.strip().lower()
+        new_donation = int(input("\nEnter the 'Amount' donated >"))
+        if donor_name not in donors:
+            donors[donor_name] = [new_donation]
+            print(f'\n Donation amount of "{new_donation}" has been recorded for {donor_name.title()}!!\n')

```

Comment:
Since this message seems to be a duplicate of the message printed in the event of an existing user, you may want to consider having  a function handle the printing so you can call it for both functions, instead of having the same code copied.